.ci/java-versions-aarch64.properties

```diff
@@ -4,4 +4,4 @@
 # build and test Elasticsearch for this branch. Valid Java versions
 # are 'java' or 'openjdk' followed by the major release number.
 
-ES_BUILD_JAVA=jdk17
+ES_BUILD_JAVA=jdk21
```

Comment:
I think this can just be openjdk21 as well. I don't think we need separate ones for arm vs x64 anymore.

Reply:
Oh! Good catch Mark. Let’s do that in a follow up.